build/ci/cloudbuild.push.yaml

```diff
@@ -0,0 +1,109 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  - id: update-repo-version
+    name: "gcr.io/cloud-builders/git"
+    entrypoint: /bin/bash
+    args:
+      - -c
+      - |
+        set -e
+        gcloud source repos clone deployment /tmp/deployment --project=datcom-ci
+        cd /tmp/deployment
+        ./scripts/update_external_repo_version.sh $REPO_NAME $SHORT_SHA
+    waitFor: ["-"]
+
+  - id: push-web-server
+    name: gcr.io/cloud-builders/docker
+    entrypoint: "bash"
+    args:
+      - -c
+      - |
+        set -e
+        docker build -f build/web_server/Dockerfile \
+          -t gcr.io/datcom-ci/datacommons-website:$SHORT_SHA \
+          -t gcr.io/datcom-ci/datacommons-website:latest \
+          .
+        docker push gcr.io/datcom-ci/datacommons-website:$SHORT_SHA
+        docker push gcr.io/datcom-ci/datacommons-website:latest
+
+  - id: push-nl-server
+    name: gcr.io/cloud-builders/docker
+    entrypoint: "bash"
+    args:
+      - -c
+      - |
+        set -e
+        docker build -f build/nl_server/Dockerfile \
+          -t gcr.io/datcom-ci/datacommons-nl:$SHORT_SHA \
+          -t gcr.io/datcom-ci/datacommons-nl:latest \
+          .
+        docker push gcr.io/datcom-ci/datacommons-nl:$SHORT_SHA
+        docker push gcr.io/datcom-ci/datacommons-nl:latest
+    waitFor: ["-"]
+
+  - id: push-nodejs-server
+    name: gcr.io/cloud-builders/docker
+    entrypoint: "bash"
+    args:
+      - -c
+      - |
+        set -e
+        docker build -f build/nodejs_server/Dockerfile \
+          -t gcr.io/datcom-ci/datacommons-nodejs:$SHORT_SHA \
+          -t gcr.io/datcom-ci/datacommons-nodejs:latest \
+          .
+        docker push gcr.io/datcom-ci/datacommons-nodejs:$SHORT_SHA
+        docker push gcr.io/datcom-ci/datacommons-nodejs:latest
+    waitFor: ["-"]
+
+  - id: push-website-cron-testing
+    name: gcr.io/cloud-builders/docker
+    entrypoint: "bash"
+    args:
+      - -c
+      - |
+        set -e
+        docker build -f build/website_cron_testing/Dockerfile \
+          -t gcr.io/datcom-ci/website-cron-testing:$SHORT_SHA \
+          -t gcr.io/datcom-ci/website-cron-testing:latest \
+          .
+        docker push gcr.io/datcom-ci/website-cron-testing:$SHORT_SHA
+        docker push gcr.io/datcom-ci/website-cron-testing:latest\
+    waitFor: ["-"]
+
+  - id: update version
```

Comment:
maybe add some comments for this one (and differentiate with update-repo-version)

Reply:
Renamed and added comments.